resources/migrations/000_migrations.yaml

```diff
@@ -3649,6 +3649,7 @@ databaseChangeLog:
   - changeSet:
       id: 56
       author: wwwiiilll
+      comment: 'Added 0.25.0'
```

Comment:
Yeah, I was thinking it's a lot easier than to having to go look at history in GitHub